### PR TITLE
S/dev merge

### PR DESCRIPTION
**Pull request by silabs-krdosvik**
Merge.
Formal passes,
hello-world passes
ci check passes except pma cfg 2 tests

### CODE REVIEW

**Review by silabs-robin — COMMENTED** [review]
(Darn, forgot to un-pend my comments.)

File: bin/merge.sh
Comment (silabs-krdosvik) [reply bin/merge.sh:1]: Yes!

**Review by silabs-robin — COMMENTED** [review]
Most files are from `cv32e40s/`, not very relevant.
All files from `bin/` and `lib/` doen't look like they will cause any problems, only progress.
Will approve when all conversations are resolved.

File: bin/merge.sh
Comment (silabs-krdosvik) [reply bin/merge.sh:1]: FIXED (this should shortly be updated as again after new merge from cv32e40s to cv32e40x-dv)

**Review by silabs-robin — APPROVED** [review]
Approving based on:
Reasoning given in comments.

File: bin/merge.sh
Comment (silabs-krdosvik) [reply bin/merge.sh:1]: > On which x-dv?
Yes, this is the newly merged cv32e40x-dv.

> was for it to be stable 
True. Yea... But yes, we are mostly active, so fine for now, hehe..

**Pull request merged**
